segmentation/utils/spatial_analysis_utils.py

```diff
@@ -255,27 +261,27 @@ def calculate_enrichment_stats(close_num, close_num_rand):
     return stats_xr
 
 
-def compute_neighbor_counts(fov_data, dist_matrix, distlim, self_neighbor=True,
+def compute_neighbor_counts(current_fov_data, dist_matrix, distlim, self_neighbor=True,
                             cell_label_col="cellLabelInImage"):
     """Calculates the number of neighbor phenotypes for each cell. The cell counts itself as a neighbor.
 
     Args:
-        fov_data: data for the current fov, including the cell labels, cell phenotypes, and cell phenotype ID
-        dist_matrix: cells x cells matrix with the euclidian
+        current_fov_data (pandas df): data for the current fov, including the cell labels, cell phenotypes, and cell phenotype ID
```

Comment:
This should be current_fov_neighborhood_data to match the name in the main function

Reply:
When you call this function, the input array is called current_fov_neighborhood_data, so I think we should give it the same name inside of the function: https://github.com/angelolab/segmentation/blob/3f7e8db3e63600574e314e1c604d38224437e804/segmentation/utils/spatial_analysis.py#L249